### ISSUE TITLE
Grbl $30 and $31 values

### ISSUE THREAD

**Issue by gnurc** [kirilnedev/grbl-rc-pwm#1]
Hello,

Thanks for this project works to me, but not very precise. That values you use for $30 and $31?

I'm using 12000 for $30 and 0 for $31 but it goes a little crazy.

Best regards

**Comment by kirilnedev** [comment]
You're right... I figured out it has some serious problems, also with setting the speed to 0% and 100%. I will try to find some time and fix it. Thanks for reporting

**Comment by gnurc** [comment]
https://www.youtube.com/watch?v=nxm5qQQCp8g&feature=emb_logo

Thank you,

Sometimes it stops shortly after starting a job at bCNC and the bit breaks. I don't know why, then after a while it turns on again.
